pkg/kubelet/cm/dra/manager.go

```diff
@@ -97,6 +97,18 @@ func (m *ManagerImpl) prepareContainerResources(pod *v1.Pod, container *v1.Conta
 				return fmt.Errorf("failed to fetch ResourceClaim %s referenced by pod %s: %+v", claimName, pod.Name, err)
 			}
 
+			// Check if pod is in the ReservedFor for the claim
```

Comment:
There is a `resourceclaim.IsReservedForPod` function that you can use here.

Reply:
done